src/Compilers/CSharp/Portable/CSharpResources.Designer.cs

```diff
@@ -8567,6 +8567,15 @@ internal class CSharpResources {
         }
         
         /// <summary>
+        ///   Looks up a localized string similar to The source does not appear to be valid C# code..
+        /// </summary>
+        internal static string ERR_WrongLanguage {
```

Comment:
:bulb: If you push to the gafter/fix2771 branch, this pull request will be automatically updated. Any comments made on the diff for the pull request (including these) will not be lost, but they'll be marked "out of date" in the view. This works whether you add a new commit or amend an existing one (which means it also works if you add a new commit, and then squash the commits later before merging the pull request).